src/flowcean/transforms/flatten.py

```diff
@@ -63,11 +63,7 @@ def transform(self, data: pl.DataFrame) -> pl.DataFrame:
 
             # Figure out how "long" the feature is and how many new columns
             # need to be added
-            row_lengths = data.select(
-                pl.col(feature)
-                .list.eval(pl.first().struct.field("value"))
-                .list.len()
-            ).unique()
```

Comment:
Was this change intended?

Reply:
That's a leftover artefact from trying to figure out if the error was in our code or in the polars. In the process, I realised that the check was overly complicated (we don't really need to unpack the strucst to know how many there are). So the change is "good", it just doesn't necessarily belong in this PR.